Dockerfile

```diff
@@ -1,72 +1,16 @@
-FROM alpine:3.8
+from node:latest
```

Comment:
Is it possible to pin it to a specific major version (e.g. 12.x)?  I generally test CyTube against the current LTS version of node; sometimes the newer versions introduce regressions and/or incompatibilities with certain C++ modules that then need to be updated to support it.